ambry-server/src/integration-test/java/com.github.ambry.server/ServerTestUtil.java

```diff
@@ -2003,7 +2003,7 @@ private static void checkReplicaTokens(MockClusterMap clusterMap, DataNodeId dat
           DataInputStream dataInputStream = new DataInputStream(crcStream);
           try {
             short version = dataInputStream.readShort();
-            assertEquals(0, version);
+            assertEquals(1, version);
 
             System.out.println("setToCheck" + setToCheck.size());
```

Comment:
remove this?

Reply:
this was from before, but fixed nonetheless!